ttl/hbp_dataset_attributes_ontology.ttl

```diff
@@ -0,0 +1,50 @@
+### WARNING ###
+### THIS ONTOLOGY IS TEMPORARY DO NOT USE IDENTIFIERS OUTSIDE DEV
+### WARNING ###
+@prefix : <http://www.hbp.FIXME.org/hbp_dataset_attributes_ontology/> .
```

Comment:
why _dataset_ attributes? We also have them in resources and in the future they could be added in even more locations.